### DISCUSSION TITLE
Feature Request: Restore Support for Dynamic Kibana Dashboard URLs in ElastAlert2

### DISCUSSION THREAD

**Discussion by naasir-ansar** [jertel/elastalert2#1577] [Ideas]
In previous versions of ElastAlert, we had functionality to generate dynamic Kibana dashboard URLs directly in alerts (referenced in the [documentation](https://elastalert.readthedocs.io/en/latest/ruletypes.html?highlight=kibana4)). This was highly beneficial as it allowed alerts to link users directly to relevant dashboards for quick investigation. Currently, ElastAlert2 only supports dynamic URLs for Kibana’s Discover app, which limits usability.

Feature Request:
	•	Please consider restoring the option to generate dynamic Kibana dashboard URLs in alerts, similar to how it functioned in earlier versions.

Questions:
	•	Could you clarify why support for dynamic dashboard URLs was removed in ElastAlert2?
	•	Are there any existing limitations or technical challenges in reintroducing this feature?

This feature would enhance usability for teams using ElastAlert2 alongside Kibana dashboards for monitoring and alert investigations.

Thank you for considering this request!

**Comment by nsano-rururu** [comment]
@naasir-ansar 
This feature was removed when elasticsearch 8 was supported, but it had not worked for quite some time.
If this is a necessary feature, we welcome pull requests.
https://github.com/jertel/elastalert2/pull/744#issuecomment-1059268470
https://github.com/jertel/elastalert2/pull/744#issuecomment-1059354075
